pkg/operator/apis/monitoring/v1/http_types_test.go

```diff
@@ -24,10 +24,13 @@ import (
 	"github.com/GoogleCloudPlatform/prometheus-engine/pkg/secrets"
 )
 
-type secretNamespaceTestCase struct {
+type secretNamespacePodMonitoringTestCase struct {
```

Comment:
nit: if the objects are the same, I would just reuse the same struct.

Reply:
42b36b7d419f0ac1a7f60942ac1e61ef026eda21